css/style.css

```
@@ -815,3 +1619,152 @@ input[type="radio"].visually-hidden {
   fill-opacity: 0.1;
 }
 
+/* modal */
+
+.modal {
+  position: fixed;
+  display: none;
+  top: 10%;
+  right: 0;
+  bottom: auto;
+  left: 0;
+  margin: auto;
+  box-shadow: 0px 10px 16px rgba(0, 0, 0, 0.15);
+  background-color: var(--basic-white);
+  width: 960px;
+}
+
+.modal--active {
+  display: block;
+}
+
+.modal__close {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  border: none;
+  width: 55px;
+  height: 55px;
+  border-radius: 50%;
+  background-color: var(--special-yellow-dark-opaque);
+  background-image: url("../img/icons/icon_cross.svg");
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+  cursor: pointer;
+}
+
+.modal__close:hover {
+  background-color: var(--special-yellow-dark);
+}
+
+.modal__close:active {
+  background-color: var(--special-yellow-dark-more-opaque);
+}
+
+/* map */
+
+.modal--map {
+  width: 960px;
+  height: 573px;
+  background-image: url("../img/map-full.png");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.modal__interactive-map {
+  border: none;
+}
+
+/* feedback */
+
+.feedback {
+  display: grid;
+  grid-template-columns: 360px 360px;
```

гибче, не стоит хардкодить ширину
```suggestion
  grid-template-columns: 1fr 1fr;
```